common/src/main/java/com/microsoft/identity/common/internal/ui/webview/AzureActiveDirectoryWebViewClient.java

```diff
@@ -370,32 +447,29 @@ public void run() {
         return true;
     }
 
-    private boolean processInvalidUrl(@NonNull final WebView view, @NonNull final String url) {
-        final String lowerCaseUrl = url.toLowerCase(Locale.US);
-        if (isBrokerRequest(getActivity().getIntent())
-                && url.startsWith(AuthenticationConstants.Broker.REDIRECT_PREFIX)) {
-            Logger.error(TAG, "The RedirectUri is not as expected.", null);
-            Logger.errorPII(TAG, String.format("Received %s and expected %s", url, mRedirectUrl), null);
-            returnError(ErrorStrings.DEVELOPER_REDIRECTURI_INVALID,
-                    String.format("The RedirectUri is not as expected. Received %s and expected %s", url,
-                            mRedirectUrl));
-            view.stopLoading();
-            return true;
-        }
+    private boolean processRedirectUriCheck(@NonNull final WebView view, @NonNull final String url) {
```

Comment:
This is a dead-end method, right?  We should name it that way.

Reply:
processInvalidUrl(url) is supposed to be the dead-end method.

Reply:
What I'm saying is that there's no need for it to return.  If you call this method, you know what it's going to return.  Same with all of these, for the most part.  This is a minor comment, but it means that somebody reading the code might think there's the possibility it could return something dependent on its input.  It cannot, and then they have to go read the method to make that determination.